### PR TITLE
Update the instructions to disable ASM, following ENT customer feedback

### PR DESCRIPTION
**Pull request by cyrbouchiat**
Adding instructions to disable specific services (PHP and RC enabled services)

### Merge instructions

- [ ] Please merge after reviewing

### CODE REVIEW

File: content/en/security/application_security/threats/library_configuration.md [hunk 1]
```diff
@@ -100,6 +100,10 @@ See [APM Data Security][3] for information about other mechanisms in the Datadog
 
 To disable ASM, remove the `DD_APPSEC_ENABLED=true` environment variable from your application configuration. Once it's removed, restart your service.
 
+In case no `DD_APPSEC_ENABLED=true` environment variable is set for your service:
```
Comment (estherk15) [inline content/en/security/application_security/threats/library_configuration.md:103]:
```suggestion
If no `DD_APPSEC_ENABLED=true` environment variable is set for your service:
```

File: content/en/security/application_security/threats/library_configuration.md [hunk 2]
```diff
@@ -100,6 +100,10 @@ See [APM Data Security][3] for information about other mechanisms in the Datadog
...
+* If it's a PHP service: explicitely set the environment variable to `DD_APPSEC_ENABLED=false` and restart your service
```
Comment (estherk15) [inline content/en/security/application_security/threats/library_configuration.md:104]:
```suggestion
* If it's a PHP service: explicitly set the environment variable to `DD_APPSEC_ENABLED=false` and restart your service
```

File: content/en/security/application_security/threats/library_configuration.md [hunk 3]
```diff
@@ -100,6 +100,10 @@ See [APM Data Security][3] for information about other mechanisms in the Datadog
...
+* If ASM was activated in one-click: go to [ASM service configuration][8], click on your service and press 'Deactivate'
```
Comment (estherk15) [inline content/en/security/application_security/threats/library_configuration.md:105]:
```suggestion
* If ASM was activated in one-click: go to [ASM service configuration][8], click on your service and then click **Deactivate**.
```

**Review by estherk15 — APPROVED** [review]

**Comment by aliciascott** [comment]
Secondary review approved, with approval to merge from @cyrbouchiat

**Pull request merged**